bliss/catalog.py

```diff
@@ -61,6 +63,70 @@ def __init__(self, tile_slen: int, d: Dict[str, Tensor]):
         self.batch_size, self.n_tiles_h, self.n_tiles_w, self.max_sources = self.locs.shape[:-1]
         super().__init__(**d)
 
+    def _test_tensor_all_close(self, left, right):
+        return torch.allclose(left, right, atol=1e-4)
```

Comment:
Will the checks pass with a smaller atol? 1e-4 is a bit lax for some values

Reply:
I'll fix it in the next commit

Reply:
When testing these tensors, the tensors of fluxes sometimes mismatch and `atol=1e-4` is the minimal tolerance that can suppress this warning.

Reply:
I see. Is that because the fluxes really large? If so, maybe we could specify `rtol` too?

Reply:
i'll try it

Reply:
Oh it looks like rtol is set by default to a pretty small value. It must be that the fluxes were very large.